core/src/main/java/com/predic8/membrane/core/interceptor/balancer/LoadBalancingInterceptor.java

```diff
@@ -156,7 +156,7 @@ private void setFailOverNodes(Exchange exc, Node dispatchedNode) {
     }
 
     private void updateDispatchedNode(Exchange exc) {
-        Node n = exc.getProperty("dispatchedNode", Node.class);
+        Node n = exc.getPropertyOrNull("dispatchedNode", Node.class);
```

Comment:
same here: when `getPropertyOrNull` returns `null`, this will cause a `NullPointerException` in the following line.